frontend/cavityWidget.py

```diff
@@ -8,7 +8,7 @@
 from qtpy.QtCore import Property as qtProperty, QRect, Qt, Slot
 
 from cavityFaultDisplay import CavityFaultDisplay
-from lcls_tools.common.pydm_tools.displayUtils import showDisplay
+from lcls_tools.common.frontend.display.util import showDisplay
```

Comment:
This line is importing common.frontend.display.util. We should change this to match utils then.